airflow/docker.go

```diff
@@ -663,3 +679,46 @@ func checkServiceState(serviceState, expectedState string) bool {
 	scrubbedState := strings.Split(serviceState, " ")[0]
 	return scrubbedState == expectedState
 }
+
+func startDocker() error {
+	buf := new(bytes.Buffer)
+	err := cmdExec(DockerCmd, buf, buf, "ps")
+	if err != nil {
+		// open docker
+		fmt.Println("\nIt looks like the docker engine is not running")
+		err = cmdOpenExec(OpenCmd, buf, os.Stderr, "-a", "docker")
+		if err != nil {
+			return err
+		}
+		fmt.Println("\nIf you don't see docker desktop starting exit this command and start it manually")
+		fmt.Println("If you don't have docker desktop installed install it here: https://www.docker.com/products/docker-desktop/")
+		fmt.Println("If you are using colima or another docker alternative you will have to start the engine manually")
+		fmt.Println("\nStarting up the docker engine…")
+		// poll for docker
+		err = waitForDocker(buf)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func waitForDocker(buf io.Writer) error {
```

Comment:
```suggestion
func waitForDocker() error {
    buf := new(bytes.Buffer)
```
nitpick: `buf` is not a required argument to be passed, since that's not being used after anywhere in the caller method.